mobile/lib/pages/editing/crop.page.dart

```diff
@@ -0,0 +1,185 @@
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.done_rounded, color: Colors.white, size: 24),
+            onPressed: () async {
+              Image croppedImage = await _cropController.croppedImage();;
+              Navigator.push(
+                context,
+                MaterialPageRoute(
+                  builder: (context) => EditImagePage(imageSource: croppedImage),
+                ),
+              );
+            },
+          ),
+        ],
+      ),
+      body: Column(
+        children: [
+          Container(
+            padding: const EdgeInsets.only(top: 20),
+            width: MediaQuery.of(context).size.width * 0.8,
+            height: MediaQuery.of(context).size.height * 0.6,
+            child: CropImage(
+              controller: _cropController,
+              image: widget.image,
+              gridColor: Colors.white,
+            ),
+          ),
+          Expanded(
+            child: Container(
+              width: double.infinity,
+              decoration: const BoxDecoration(
+                color: Colors.black,
+                borderRadius: BorderRadius.only(
+                  topLeft: Radius.circular(20),
+                  topRight: Radius.circular(20),
+                ),
+              ),
+              child: Center(
+                  child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Padding(
+                    padding:
+                        const EdgeInsets.only(left: 20, right: 20, bottom: 10),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        IconButton(
+                          icon: const Icon(Icons.rotate_left,
+                              color: Colors.white),
+                          onPressed: () {
+                            _cropController.rotateLeft();
+                          },
+                        ),
+                        IconButton(
+                          icon: const Icon(Icons.rotate_right,
+                              color: Colors.white),
+                          onPressed: () {
+                            _cropController.rotateRight();
+                          },
+                        ),
+                      ],
+                    ),
+                  ),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: <Widget>[
+                      Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          IconButton(
+                              icon: Icon(Icons.crop_free_rounded,
+                                  color: _selectedAspectRatio == null
+                                      ? Colors.blue
+                                      : Colors.white),
```

Comment:
Can we use some Immich colors here? I forget where our color pallette is. Maybe Alex remembers?

Reply:
Yeah I am not sure about the pallette as well, but whole app is using Colors.indigo as their color so I switched to it `color: _selectedAspectRatio == aspectRatio ? Colors.indigo : Colors.white,`